index.js

```diff
@@ -5,6 +5,7 @@
  * package tree, they will be prompted to donate.
  */
 const authors = {
+  smilerryan: 'https://paypal.me/smilerryanyt',
```

Comment:
Did you typo your npm username? There's no user with that name: https://www.npmjs.com/~smilerryan